lms/djangoapps/teams/views.py

```diff
@@ -106,6 +112,24 @@ def is_feature_enabled(course):
     return settings.FEATURES.get('ENABLE_TEAMS', False) and course.teams_enabled
 
 
+def has_discussion_privileges(user, course_id):
```

Comment:
What do you think about moving this method to django_comment_common to keep more of the logic there? I was actually imagining that it would be in utils (where get_role_ids lives). And I would think it should have unit test coverage.